impress-org/givewp
# Refactor: use wp scripts to compile Gutenberg blocks

## Description



Initially, I added [these lines](https://github.com/impress-org/givewp/blob/develop/webpack.mix.js#L85-L88) in the webpack settings file to solve [this other bug](https://github.com/impress-org/givewp/pull/7069) and I didn't notice that it prevent the Donor Dashboard to get loaded.

So, to fix the initial problem and keep the Donor Dashboard working, was necessary to compile the scripts used in the Gutenberg blocks with WP Scripts.

## Affects



- Multi-Form Goal Block > Progress Bar > Filters
- Donor Dashboard

## Testing Instructions



1. Create a new page;
2. Add the multi-form goal block;
3. Click on the progress bar;
4. On the right side, click on Filters and Filter by form;
5. No more errors should be displayed.
6. Access the Donor Dashboard, it should load without problems.

## Pre-review Checklist
